core/src/main/scala/kafka/metrics/KafkaMetricsConfig.scala

```diff
@@ -22,16 +22,33 @@ package kafka.metrics
 
 import kafka.utils.{VerifiableProperties, CoreUtils}
 
+object KafkaMetricsConfig {
+
+  val KafkaMetricsReporterClassesProp = "kafka.metrics.reporters"
```

Comment:
Would it be more readable and consistent with existing code style by moving these variables (property name, doc and default value) to KafkaConfig.java?